images/ansible/roles/kubernetes/templates/etc/systemd/system/kubelet.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=kubelet: The Kubernetes Node Agent
+Documentation=http://kubernetes.io/docs/
+
+[Service]
+ExecStart=/usr/bin/kubelet
+Restart=always
+StartLimitInterval=0
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
We shouldn't need this systemd unit file, it should be managed by the kubelet/kubeadm packages.